devices/tuya.js

```diff
@@ -381,6 +381,19 @@ module.exports = [
         },
         exposes: [e.battery(), e.illuminance_lux(), e.battery_low()],
```

Comment:
Given that the only difference is the battery and battery_low expose, try with removing the `S-LUX-ZB TS0601` and change this line to.

```js
    exposes: (device, options) => {
        if (device.manufacturerName === '_TZE200_pisltm67') {
            return [e.illuminance_lux()];
        } else {
            return [e.battery(), e.illuminance_lux(), e.battery_low()];
        }
    },
```

Note: make sure to run at least z2m 1.23.0